src/models/AirImpurityDataModel.ts

```diff
@@ -0,0 +1,11 @@
+import { type AirImpurityData } from '@/mocks/SensorData';
+
+export class AirImpurityDataModel implements AirImpurityData {
```

Comment:
I don't think we need to create models for simple objects that hold data. If these objects have no methods, it's better not to add more complexity and have just a parsed JSON object.